media/open_zim.ksy

```diff
@@ -0,0 +1,409 @@
+        value: _parent.url_pointer_list[index].entry
+  str_list:
+    seq:
+      - id: items
+        type: strz
+        repeat: until
+        repeat-until: _ == ""
+  directory_entry_ptr:
+    seq:
+      - id: ptr
+        type: u8
+    instances:
+      entry:
+        pos: ptr
+        type: directory_entry
+    types:
+      directory_entry:
+        seq:
+          - id: mime_idx
+            -orig-id: mimetype
+            type: u2
+            doc: MIME type number as defined in the MIME type list
+          - id: parameter_size
+            -orig-id: parameter_len
+            type: u1
+            doc: (not used) length of extra paramters
+          - id: namespace
+            type: s1
+            enum: namespace
+            doc: defines to which namespace this directory entry belongs
+          - id: revision
+            type: u4
+            doc: (optional) identifies a revision of the contents of this directory entry, needed to identify updates or revisions in the original history
+          - id: body
+            type:
+              switch-on: mime_idx == 0xffff
+              cases:
+                true: redirect
+                _: article
+          - id: url
+            type: strz
+            doc: string with the URL as refered in the URL pointer list
+          - id: title
+            type: strz
+            doc: string with an title as refered in the Title pointer list or empty; in case it is empty, the URL is used as title
+          - id: parameter
+            size: parameter_size
+            doc: (not used) extra parameters
+        instances:
+          mime:
+            value: _root.mime_list.items[mime_idx]
+            if: mime_idx != 0xffff
+
+        types:
+          article:
+            seq:
+              - id: cluster_index
+                -orig-id: cluster_number
+                type: u4
+                doc: cluster number in which the data of this directory entry is stored
+              - id: blob_index
+                -orig-id: blob_number
+                type: u4
+                doc: blob number inside the compressed cluster where the contents are stored
+            instances:
+              cluster:
+                value: _root.cluster_pointer_list[cluster_index].cluster
+              data:
+                value: cluster.blobs[blob_index]
+          redirect:
+            seq:
+              - id: redirect_index
+                -orig-id: redirect_index
+                type: u4
+                doc: pointer to the directory entry of the redirect target
+        enums:
+          namespace:
+            0x2D: # "-"
+              id: layout
+              doc: eg. the LayoutPage, CSS, favicon.png (48x48), JavaScript and images not related to the articles
+            0x41: # "A"
+              id: article
+              doc: articles - see Article Format
+            0x42: # "B"
+              id: article_meta_data
+              doc: article meta data - see Article Format
```

Comment:
There is also a `H` namespace, which I found in a test file (`nomadic-changthang` which can be found on https://wiki.kiwix.org/wiki/Content_in_all_languages ) but I have no idea what that namespace actually means or represents.